Add gloo to docs (#886)
knative/docs: install/Knative-custom-install.md

## Installing Istio

> > Note: [Gloo](https://gloo.solo.io/) now provides an alternative installation option

Reviewer: ```suggestion
> Note: [Gloo](https://gloo.solo.io/) is available as an alternative to Istio.
```

> > for those wishing to install Knative without Istio. Note that Knative-Eventing

Reviewer: Global comment: "Knative-Eventing" --> "the Knative Eventing component".

> > for those wishing to install Knative without Istio. Note that Knative-Eventing

Reviewer: ```suggestion
> Gloo is not currently compatible with the Knative Eventing component.
```

> > is not supported by Gloo at this time. [Click here](Knative-with-Gloo.md) to follow the

Reviewer: ```suggestion
> See [Knative Install using Gloo](Knative-with-Gloo.md) to use Gloo instead of Istio.
```

> > Gloo-based install instructions for Knative.

Reviewer: ```suggestion
```

Reviewer: Copy changes to this note globally.
